rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1269,8 +1269,6 @@ def test_upgrade_db_35_to_36(user_data_dir):  # pylint: disable=unused-argument
         ('0x2B888954421b424C5D3D9Ce9bB67c9bD47537d12', 'ETH', 'lefteris GTC'),
         ('0xc37b40ABdB939635068d3c5f13E7faF686F03B65', None, 'yabir everywhere'),
     ]
-    cursor.execute('SELECT extra_data FROM history_events WHERE counterparty="liquity"')
-    res = cursor.fetchall()
```

Comment:
Ah, I deleted it because this was a useless query. `res` was not compared with anything.

Reply:
Don't just delete it. Check what needs to be checked instead. If it was not needed is fine but if someone forgot to check something please do

Reply:
Not sure whether there should be a check because the idea of this part of the upgrade is to remove the events for redecoding.

However, just in case, I added a check that `res` is an empty list here.